### ISSUE TITLE
Invalid syntax in `symp.obo`

### ISSUE THREAD

**Issue by althonos** [DiseaseOntology/SymptomOntology#4]
Hi !

# Errors

There are a few syntax errors in the current version `symp.obo`:

- on line 2246, the xref URL did not properly replace the `<new dbxref>` placeholder: 
   ```
   [URL:http\://www2.merriam-webster.com/cgi-bin/mwmednlm?book=Medical&va=hyperthermia<new dbxref>]
   ```
- on line 4774 and 5507, several xref URLs are merged together:
   ```
   [URL:http\://www2.merriam-webster.com/cgi-bin/mwmednlm?book=Medical&va=hemorrhage <http\://www2.merriam-webster.com/cgi-bin/mwmednlm?book=Medical&va=hemorrhage, wiki:http\://en.wikipedia.org/wiki/Gingival]
   ```
- on lines 5157 and 5186, the placeholder xref has not been replaced
   ```
   xref: ICD9CM_2005:<new dbxref>
   ```
   In particular:
      - `SYMP:0000816` is probably linking `ICD9CM:307.46` or `ICD9CM:307.47`
      - `SYMP:0000821` is probably linking to `ICD9CM:786.6`

# Suggestions

A few suggestions to improve the management of xrefs:

- xrefs to Wikipedia articles can use the `Wikipedia` prefix with only the article name, e.g. `Wikipedia:Hematemesis`.
- since URLs are valid identifiers, there is no need to prefix URLs to web resources with a `url` prefix.
- on line 5616 the URL is invalid (missing leading `h` in `http`)
- the `part_of` relationship should have an xref referring to `BFO:0000050` so that the actual BFO relation can be substituted to the locally defined one as stated in the OBO Semantcis.

**Comment by lschriml** [comment]
Hello Martin,
 thank you for pointing out these fixes in the Symptom Ontology !!

What project are you working on. ?

Edits pushed. 

Cheers,
Lynn

**Comment by althonos** [comment]
Hi Lynn,
I'm trying to improve the semantics of the OBO ontology language to improve conversion from and to the OWL standard. I developed a formal parser ([althonos/fastobo](https://github.com/althonos/fastobo)) and I'm trying to get [all the ontologies of the OBO Foundry to update to the new 1.4 syntax](https://github.com/althonos/fastobo/projects/1). I'm now working on the OBO to OWL semantics to produce a formally correct `obo2owl` converter, with Python support, and no need for a Java stack!

**Comment by althonos** [comment]
Hi Lynn,
this issue has not been fixed completely: `SYMP:0000821` still contains an invalid xref (`xref: ICD9CM_2005:<new dbxref>`) in the latest commit on `master` branch.
